robot/packages/cloud_package_manager.go

```diff
@@ -162,14 +116,12 @@ func (m *cloudManager) Sync(ctx context.Context, packages []config.PackageConfig
 	newManagedPackages := make(map[PackageName]*managedPackage, len(packages))
 
 	for idx, p := range packages {
-		select {
-		case <-ctx.Done():
-			return multierr.Append(outErr, ctx.Err())
-		default:
+		if err := ctx.Err(); err != nil {
+			return multierr.Append(outErr, err)
 		}
 
 		if err := p.Validate(""); err != nil {
-			m.logger.Debugf("package config validation error; skipping", "package", p.Name, "error", err)
+			m.logger.Errorw("package config validation error; skipping", "package", p.Name, "error", err)
```

Comment:
skipping a package should be an error that the user sees. Otherwise components that rely on packages will produce unintuitive error messages.

Reply:
Agreed, thank you for changing this!

Reply:
Ah sorry; I made this a debug log but should've done an error log. Module and process config validation logs are both errors.